Gemfile.lock

```diff
@@ -383,10 +388,6 @@ GEM
       connection_pool (~> 2.2)
     netrc (0.11.0)
     nio4r (2.5.8)
-    nokogiri (1.13.9-arm64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.13.9-x86_64-darwin)
-      racc (~> 1.4)
```

Comment:
Probably nothing in this - just trying to work out why it's happening. Is it just because the previous updater of Gemfile.lock was on Mac and you're not? Or if we need to ship all three as options does that mean that Bundle can install the correct version for the architecture it's being run on (as opposed to trying to install all three and failing)?

If the latter, maybe we can leave these in?

Reply:
I have a feeling that this was here (or absent) because the previous updater was on mac and I am not.
For production and Docker the only one needed is the Linux one.  The rest could stay or go without much fuss.

My understanding is that Bundler is smart enough to ignore the versions that it doesn't need.